### ISSUE TITLE
Fix warnings while extracting JQ archive

### ISSUE THREAD

**Issue by iamazeem** [liquidaty/zsv#204]
On Linux, there are these **unknown extended headers** warnings while extracting `jq-1.6.tar.bz2`:

```shell
$ cd /home/runner/work/zsv/zsv/build/Linux/rel/musl-gcc/external/jq-src-tmp && tar xf /home/runner/work/zsv/zsv/app/external/jq-1.6.tar.bz2
tar: Ignoring unknown extended header keyword 'LIBARCHIVE.xattr.com.apple.quarantine'
tar: Ignoring unknown extended header keyword 'LIBARCHIVE.xattr.com.apple.provenance'
...
```

According to this [SO thread](https://stackoverflow.com/questions/51655657/tar-ignoring-unknown-extended-header-keyword-libarchive-xattr-security-selinux), the archive needs to recompressed.
Need to test and verity this.

**Comment by iamazeem** [comment]
@liquidaty:
Compared https://github.com/liquidaty/zsv/blob/main/app/external/jq-1.6.tar.bz2 with the official release (https://github.com/jqlang/jq/releases/download/jq-1.6/jq-1.6.tar.gz) and both are identical. :thinking:
Seems like ours has been extracted and re-archived on macOS.
Shouldn't we use and store the official one?

Also, there's been new release https://github.com/jqlang/jq/releases/tag/jq-1.7.1 with security fixes.
Shouldn't we use this one?

**Comment by iamazeem** [comment]
@liquidaty: PRs created for both `jq-1.6` and `jq-1.7.1`:

- #207
- #208 

Thanks!

**Comment by iamazeem** [comment]
**UPDATE**
Closed the PR for `jq-1.7.1`.
Doesn't work with MinGW.
Locally, it compiles fine but the tests fail at runtime.
In CI, the compilation wasn't successful.

`jq-1.6` seems to work fine both locally and in the CI.
